manifests/0000_70_cluster-network-operator_04_kubeapiserver_flowschema.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1alpha1
+kind: FlowSchema
+metadata:
+  name: openshift-sdn
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+spec:
+  distinguisherMethod:
+    type: ByUser
+  matchingPrecedence: 500
+  priorityLevelConfiguration:
+    name: system
+  rules:
+  - nonResourceRules:
+    - nonResourceURLs:
+      - '*'
+      verbs:
+      - '*'
+    resourceRules:
+    - apiGroups:
+      - '*'
+      clusterScope: true
+      namespaces:
+      - '*'
+      resources:
+      - '*'
+      verbs:
+      - '*'
+    subjects:
+    - kind: ServiceAccount
+      serviceAccount:
```

Comment:
did i get the service account right?

Reply:
We have two service accounts, `sdn-controller` and `sdn`. They're both system priority.

Reply:
@squeed I added `sdn-controller` as you suggested.